common/src/main/java/com/wynntils/screens/settings/widgets/ApplyButton.java

```diff
@@ -22,7 +23,9 @@ public ApplyButton(WynntilsBookSettingsScreen screen) {
                 14,
                 Component.translatable("screens.wynntils.settingsScreen.apply"),
                 List.of(Component.translatable("screens.wynntils.settingsScreen.apply.description")
-                        .withStyle(ChatFormatting.GREEN)));
+                        .withStyle(ChatFormatting.GREEN)),
+                0,
+                McUtils.mc().screen.height);
```

Comment:
This and CloseButton don't need to worry about the mask but they are `GeneralSettingsButton`'s so they need it, when they get replaced with the side tags in the redesign it'll be fine